test/llvm_tests/FunctionMapping/LLJitMapFunctionsToJitSymbolsTests.cpp

```diff
@@ -0,0 +1,473 @@
+//
+// Created by Ciaran on 27/10/2021.
+//
+
+#include "llvm/LLJit.h"
+#include "gtest/gtest.h"
+#include "TestModelFactory.h"
+#include "rrLogger.h"
+#include "rrRoadRunnerOptions.h"
+#include "MapFunctionsToJitSymbolsTests.h"
+
+using namespace rr;
+using namespace rrllvm;
+
+
+class LLJitMapFunctionsToJitSymbolsTests : public MapFunctionsToJitSymbolsTests {
+public:
+    LLJitMapFunctionsToJitSymbolsTests()
+        : MapFunctionsToJitSymbolsTests(std::make_unique<LLJit>(LoadSBMLOptions().modelGeneratorOpt)){
+        Logger::setLevel(Logger::LOG_DEBUG);
+    };
+};
+
+TEST_F(LLJitMapFunctionsToJitSymbolsTests, CheckLibFuncPow) {
+    LLJit* llJit = (LLJit*)jit.get();
+    std::cout << llJit->emitToString() << std::endl;
+    llJit->addIRModule();
+    CheckLibFuncPow();
+}
+
+TEST_F(LLJitMapFunctionsToJitSymbolsTests, CheckLibFuncFabs){
+    CheckLibFuncFabs();
+}
+
+TEST_F(LLJitMapFunctionsToJitSymbolsTests, CheckLibFuncFabs2){
+    double expected = 5.3;
+    std::string funcName = "fabsd";
+    llvm::Function* f = jit->getModuleNonOwning()->getFunction(funcName);
+    std::cout << f << std::endl;
+//    fabsFnTy fnPtr = (fabsFnTy) jit->getModuleNonOwning()->getFunction(funcName);
```

Comment:
Looks like this test wasn't quite finished.

Reply:
Sometimes I create dud test for looking into a bug. It's quicker than needing a whole new file with its own main. This did indeed get forgotten about and is now removed.